package NYT.NQueryClient.NProto;

////////////////////////////////////////////////////////////////////////////////

message TQueryStatistics
{
    required int64 rows_read = 1;
    required int64 rows_written = 2;
    required uint64 sync_time = 3;
    required uint64 async_time = 4;
    required uint64 execute_time = 5;
    required uint64 read_time = 6;
    required uint64 write_time = 7;
    required bool incomplete_input = 8;
    required bool incomplete_output = 9;
}

////////////////////////////////////////////////////////////////////////////////

